src/web/components/BusinessContacts/BusinessContactsTable.tsx

```diff
@@ -10,19 +12,28 @@ function NoEmailContact({
 }: {
   onAddEmailContact: (form: BusinessContactForm) => Promise<void>;
 }) {
+  const [showBusinessContactDialog, setShowBusinessContactDialog] = useState<boolean>(false);
+  const onOpenChangeBusinessContactDialog = () => {
+    setShowBusinessContactDialog(!showBusinessContactDialog);
+  };
   return (
     <TableNoDataPlaceholder
```

Comment:
This feels like it can be combined into the component below.  Right now there's a bunch of duplicated code.

Reply:
You're right - cleaned up a lot